build.sbt

```diff
@@ -74,6 +74,7 @@ lazy val core = (project in file("core")).
     name := "gigahorse-core",
     libraryDependencies ++= Seq(sslConfig, reactiveStreams, slf4jApi, scalatest % Test),
     managedSourceDirectories in Compile += (sourceDirectory in Compile).value / "contraband-scala",
+    unmanagedSourceDirectories in Compile += (sourceDirectory in Compile).value / "contraband-scala",
```

Comment:
The duplication with `managedSourceDirectories` is a little suspicious to me.. Is it really ok to have it in both?